config/src/config/network_config.rs

```diff
@@ -191,6 +191,43 @@ impl NetworkConfig {
         };
         self.network_keypairs = Some(network_keypairs);
     }
+
+    /// Use this node config to build a seed peer address (using the given
+    /// `bootstrap_address` instead of the configured `self.advertised_address`).
+    pub fn build_seed_address(&self, bootstrap_address: NetworkAddress) -> Result<NetworkAddress> {
+        // TODO(philiphayes): use enum in wire crate when that gets extracted
+        let handshake_version = 0;
```

Comment:
we have a few HANDSHAKE_VERSION constant already, why not use one of them?

Reply:
there were some problems with circular dependencies when i tried to unify, so i'm going to handle extracting this in a follow up